Do not try to wake up offline SMT siblings.

-ck

---
 kernel/sched/bfs.c |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-4.7-ck5/kernel/sched/bfs.c
===================================================================
--- linux-4.7-ck5.orig/kernel/sched/bfs.c	2016-09-23 08:32:58.957732390 +1000
+++ linux-4.7-ck5/kernel/sched/bfs.c	2016-09-23 08:32:58.955732403 +1000
@@ -3544,6 +3544,8 @@ static void wake_smt_siblings(struct rq
 		struct rq *rq;
 
 		rq = cpu_rq(other_cpu);
+		if (!rq->online)
+			continue;
 		if (rq_idle(rq)) {
 			struct task_struct *p = rq->curr;
 
